src/main/index.js

```diff
@@ -1,29 +1,28 @@
-const {
+import {
   app,
   BrowserWindow,
   Menu,
   dialog,
   powerMonitor,
   shell,
-  Notification
-} = require("electron");
-const Store = require("electron-store");
-const isDev = require("electron-is-dev");
-
-const path = require("path");
-const { version } = require("../package.json");
+  Notification,
+} from "electron";
+import Store from "electron-store";
+import isDev from "electron-is-dev";
+import path from "path";
+import { version } from "../../package.json";
 
```

Comment:
A lot of this file is formatting things (trailing commas, parenthesis, etc) that probs aren't necessary but I'm not gonna nitpick over it